### ISSUE TITLE
feat: productivity research for universities

### ISSUE THREAD

**Issue by bahrmichael** [bahrmichael/factorio-tycoon#219]

**Comment by bahrmichael** [comment]
Universities can now use module.

**Comment by winex** [comment]
well, that's nice and it were introduced in v0.3.x or v0.4.0, but i was expecting some tech added (even expensive one) as the name suggests!

modules != tech, isn't it?

the actual problem is with (again, )numbers:
- you need lots of currency for only 1 science-pack (which becomes an "average-level" delivery problem)
- 1 red belt of currency supplies 7 universities at most for blue science-packs (22-24?/pack)
this just doesn't scale as player expects. i'm really afraid about supply needs for next packs (purple?)

probable solution:
- university productivity (tech) could boost output to 2, 3 or more science-packs, depending on research level
- recipes could be balanced to require more time for more advanced science-packs, so that 7-in-a-row (see above) universities still work for next ones

p.s. you need to play x1000 science cost at least once to get that feeling, really -- it's just the numbers you could tweak, so it's not that smashing

**Comment by bahrmichael** [comment]
> modules != tech, isn't it?

Right, originally I intended a boost similar to what farms have. We can revive that! I just did some cleanup and thought it was enough.

I also thought about reducing the size of the university entity. It's pretty large at this time.

Let me reopen this issue :)
